torch/nn/quantized/modules/conv.py

```diff
@@ -186,8 +186,6 @@ class Conv1d(_ConvNd):
 
     _FLOAT_MODULE = nn.Conv1d
 
-    # We are using Conv2d to run the Conv1d. For that we need to know which
-    # dimension is squeezed/unsqueezed.
     _SQUEEZE_DIM = -2  # -2 is faster than -1.
```

Comment:
what if you remove this

Reply:
Yeah this is the culprit. I kept it around since I had to call `_packed_params.unpack()` (which calls conv2d unpack) so the output weight had to be squeezed for Conv1d. Let me directly try calling quantized::conv1d_unpack op instead.